src/ai.js

```diff
@@ -13,7 +13,7 @@ import chalk from 'chalk';
 // 4. After the text is recieved , the output is written back to the file specified
 
 // Publicly available function
-export default async function promptAI(prompt, model, temperature, outputFile, needToken) {
+export default async function promptAI(prompt, model, temperature, outputFile, needToken, apiKey = "") {
```

Comment:
Instead of passing the `apiKey` as an argument to this function (which is not desired as I don't want to pass any api keys via the command-line), we can do the reading of the `TOML` file, (i.e., `.gimme_readme_config` within the file itself.

Reply:
Could you tell me how do we are going to identify the apiKey if we are not passing as a "parameter" not "argument"

Reply:
if we using .toml I don't think .toml have any `process.toml.xxx`